datalake/backend/tests/test_check_stac_metadata.py

```diff
@@ -4,17 +4,16 @@
 from hashlib import sha256, sha512
 from io import BytesIO, StringIO
 from json import dump, dumps
-from time import sleep
 from typing import Any, Dict, List, Optional, TextIO
 from unittest.mock import MagicMock, Mock, call, patch
 
 from jsonschema import ValidationError  # type: ignore[import]
-from mypy_boto3_stepfunctions import SFNClient
 from pytest import mark, raises
 from pytest_subtests import SubTests  # type: ignore[import]
 
 from ..endpoints.utils import ResourceName
-from ..processing.check_stac_metadata.task import ProcessingAssetsModel, STACSchemaValidator, main
+from ..processing.check_stac_metadata.task import ProcessingAssetsModel, STACSchemaValidator
+from ..processing.check_stac_metadata.task import main as class_under_test
```

Comment:
`main` is not a class, it's a function. Also, why rename it at import time? There's no other `main` function in this file.